docs/rbac-setup.md

```diff
@@ -0,0 +1,13 @@
+
+# RBAC Setup 
+
+If rbac is enabled, allow the service catalog to grant access for it's
+service token.
```

Comment:
@MHBauer could you add a bit here on what would have been set in the original `helm install` command, so that people know whether they enabled RBAC? It may not be obvious to a newcomer.